openml/flows/functions.py

```diff
@@ -1,38 +1,36 @@
 import xmltodict
 
 from openml._api_calls import _perform_api_call
+from . import OpenMLFlow
+from ..util import URLError
 
 
-def _check_flow_exists(name, version):
-    """Retrieves the flow id of the flow uniquely identified by name+version.
+def get_flow(flow_id, converter=None):
+    """Download the OpenML flow for a given flow ID.
 
-    Parameter
-    ---------
-    name : string
-        Name of the flow
-    version : string
-        Version information associated with flow.
+    Parameters
+    ----------
+    flow_id : int
+        The OpenML flow id.
+    """
+    try:
+        flow_id = int(flow_id)
+    except:
+        raise ValueError("Flow ID is neither an Integer nor can be "
+                         "cast to an Integer.")
 
-    Returns
-    -------
-    flow_exist : int
-        Flow id or -1 if the flow doesn't exist.
+    try:
+        return_code, flow_xml = _perform_api_call(
+            "flow/%d" % flow_id)
+    except (URLError, UnicodeEncodeError) as e:
```

Comment:
But we should check the return code? Why don't we check the return code in ``_perform_api_call`` again?